network/p2pNetwork.go

```diff
@@ -269,7 +289,16 @@ func NewP2PNetwork(log logging.Logger, cfg config.Local, datadir string, phonebo
 	}
 	log.Infof("P2P host created: peer ID %s addrs %s", h.ID(), h.Addrs())
 
-	net.service, err = p2p.MakeService(net.ctx, log, cfg, h, la, net.wsStreamHandler, pubsubMetricsTracer{})
+	// TODO: remove after consensus v41 takes effect.
+	hm := p2p.StreamHandlerMap{
+		p2p.AlgorandWsProtocolV1:  net.wsStreamHandlerV1,
+		p2p.AlgorandWsProtocolV22: net.wsStreamHandlerV22,
+	}
+	if disableV22Protocol {
```

Comment:
kludgy and open to ideas how to remove `wsStreamHandlerV22` handler from the list to test an existing (master) node connecting to a new (feature) node

Reply:
That's probably the biggest test case that needs to be covered.